hooks/useRailgunTx.tsx

```diff
@@ -73,14 +98,9 @@ const useRailgunTx = () => {
       throw err;
     }
 
-    const { maxFeePerGas, maxPriorityFeePerGas } = await provider.getFeeData();
-
-    const gasDetailsSerialized: TransactionGasDetailsSerialized = {
-      evmGasType: EVMGasType.Type2, // Depends on the chain (BNB uses type 0)
-      gasEstimateString: gasEstimateString!, // Output from gasEstimateForShieldBaseToken
-      maxFeePerGasString: maxFeePerGas!.toHexString(), // Current gas Max Fee
-      maxPriorityFeePerGasString: maxPriorityFeePerGas!.toHexString(), // Current gas Max Priority Fee
-    };
+    const gasDetailsSerialized = await getGasDetailsSerialized(
+      gasEstimateString!
```

Comment:
How do we know `gasEstimateString` will never be undefined?

Reply:
a few lines above (~L98), we check if there was an error in the gas estimation and throw if there was. was thinking this is sufficient.

Reply:
I'm not familiar enough with the railgun SDK to know for sure. Is it true that `gasEstimateString` can never be undefined when there is no error? If so this lgtm

Reply:
i've made a note in the feedback doc about this, but i feel like for now it's safe to assume